packages/Disclosure/src/disclosure.scss

```diff
@@ -213,14 +213,14 @@
     }
 
     .ids-disclosure__header-content {
-        display: block;
         margin: var(--ids-disclosure-header-content-margin);
+        display: flex;
```

Comment:
having display flex here will still not allow wrapping like we want, that's why I think display block is better here.